### PR TITLE
feature get implementation

### PR DESCRIPTION
**Pull request by oscarlopez8**

### CODE REVIEW

File: trello/api/features/functional/BoardPositive.feature [hunk 1]
```diff
@@ -2,16 +2,19 @@
...
+  Scenario: Creates a new Board with description
     Given Sets a "POST" request to "/boards/"
       | key  | value          |
       | name | newBoard       |
       | desc | newDescription |
     When Sends request
     Then Should return status code 200
     And Saves response as "BoardObject"
-    And Validates response body
+#    And Validates response body
```
Comment (Enrique-C) [inline trello/api/features/functional/BoardPositive.feature]: Uncomment this code line

File: trello/api/features/functional/CardFuntional.feature [hunk 2]
```diff
@@ -29,9 +29,12 @@ Feature: Card
     And Sends request
     Then Should return status code 200
     And Saves response as "CardObject"
-    And Validates response body
-    And Validates schema with "card_schema.json"
+    #And Validates response body
+    #And Validates schema with "card_schema.json"
```
Comment (Enrique-C) [inline trello/api/features/functional/CardFuntional.feature]: Uncomment this code line

File: trello/api/features/functional/CardFuntional.feature [hunk 3]
```diff
@@ -29,9 +29,12 @@ Feature: Card
...
     # Post condition
-    And Sets a "DELETE" request to "/boards/(BoardObject.id)"
+    And Sets a "GET" request to "/cards/CardObject.id"
+    And Sends request
+    And Should return status code 200
```
Comment (Enrique-C) [inline trello/api/features/functional/CardFuntional.feature:36]: They are not post conditions

File: trello/api/features/functional/ListFunctional.feature [hunk 4]
```diff
@@ -8,16 +8,20 @@ Feature: List
...
+      | idBoard | (BoardObject.id) |
       | pos     | top              |
     And Sends request
     Then Should return status code 200
-    And Validates response body
-    And Validates schema with "list_schema.json"
-    And Sets a "DELETE" request to "/board/boardObject.id"
+    And Saves response as "ListObject"
+    #And Validates response body
+    #And Validates schema with "list_schema.json"
```
Comment (Enrique-C) [inline trello/api/features/functional/ListFunctional.feature]: Uncomment those code lines

File: trello/api/features/smoke/BoardSmoke.feature [hunk 5]
```diff
@@ -23,6 +26,9 @@ Feature: Board's Test
     And Sends request
     And Should return status code 200
     And Saves response as "BoardObject"
+    And Sets a "GET" request to "/boards/BoardObject.id"
+    And Sends request
+    And Should return status code 200
```
Comment (Enrique-C) [inline trello/api/features/smoke/BoardSmoke.feature]: Review those lines of code, you are testing a deletion not a creation

File: trello/api/features/smoke/ListSmoke.feature [hunk 6]
```diff
@@ -8,15 +8,19 @@ Feature: Manages board's list
...
-      | idBoard | (boardObject.id) |
+      | idBoard | (BoardObject.id) |
     And Sends request
     Then Should return status code 200
-    And Validates response body
-    And Validates schema with "list_schema.json"
-    And Sets a "DELETE" request to "/board/(boardObject.id)"
+    And Saves response as "ListObject"
+#    And Validates response body with "expected_board_body.json"
+#    And Validates schema with "list_schema.json"
```
Comment (Enrique-C) [inline trello/api/features/smoke/ListSmoke.feature]: Uncomment those code lines

**Review by Enrique-C — APPROVED** [review]

File: trello/api/features/acceptance/BoardAcceptance.feature [hunk 7]
```diff
@@ -29,20 +32,23 @@ Feature: Board
...
+    When Sets a "PUT" request to "/boards/BoardObject.id/members"
       | key   | value   |
       | email | (email) |
     And Sends request
     Then Should return status code 200
+    And Sets a "GET" request to "/boards/BoardObject.id/members"
+    And Sends request
+    And Should return status code 200
     And Validates response body
     And Validates schema with "board_schema.json"
```
Comment (Amet015) [inline trello/api/features/acceptance/BoardAcceptance.feature]: please insert the get validator in the line 46

File: trello/api/features/acceptance/BoardAcceptance.feature [hunk 8]
```diff
@@ -74,14 +80,18 @@ Feature: Board
       | color | yellow      |
     And Sends request
     Then Should return status code 200
+    And Saves response as "LabelObject"
+    And Sets a "GET" request to "/labels/LabelObject.id"
+    And Sends request
+    And Should return status code 200
```
Comment (Amet015) [inline trello/api/features/acceptance/BoardAcceptance.feature:88]: please insert the get validator in the line 89

File: trello/api/features/acceptance/BoardAcceptance.feature [hunk 9]
```diff
@@ -98,10 +108,9 @@ Feature: Board
     And Sends request
```
Comment (Amet015) [inline trello/api/features/acceptance/BoardAcceptance.feature]: please the update the line 107 (website)

File: trello/api/features/functional/BoardFunctional.feature [hunk 10]
```diff
@@ -42,13 +42,16 @@ Feature: Board
       | prefs_cardAging       | pirate                 |
     And Sends request
     Then Should return status code 200
-    And Validates response body
+    #And Validates response body
```
Comment (Enrique-C) [inline trello/api/features/functional/BoardFunctional.feature]: Uncomment this code line

File: trello/api/features/smoke/BoardSmoke.feature [hunk 11]
```diff
@@ -27,3 +30,6 @@ Feature: Board's Test
     When Sets a "DELETE" request to "/boards/BoardObject.id"
     Then Sends request
     And Should return status code 200
```
Comment (Enrique-C) [inline trello/api/features/smoke/BoardSmoke.feature]: It would be Then

File: trello/api/features/smoke/BoardSmoke.feature [hunk 12]
```diff
@@ -27,3 +30,6 @@ Feature: Board's Test
...
+    And Sets a "GET" request to "/boards/BoardObject.id"
+    And Sends request
+    And Should return status code 404
```
Comment (Enrique-C) [inline trello/api/features/smoke/BoardSmoke.feature:36]: It needs schema validator and body validator

File: trello/api/features/smoke/BoardSmoke.feature [hunk 13]
```diff
@@ -28,8 +28,10 @@ Feature: Board's Test
     And Saves response as "BoardObject"
     #Post Condition
     When Sets a "DELETE" request to "/boards/BoardObject.id"
-    Then Sends request
-    And Should return status code 200
+    And Sends request
+    Then Should return status code 200
+    And Validates response body with ""
```
Comment (Enrique-C) [inline trello/api/features/smoke/BoardSmoke.feature]: Please, delete this at the last  --> ""

File: trello/api/features/smoke/BoardSmoke.feature [hunk 14]
```diff
@@ -28,8 +28,10 @@ Feature: Board's Test
     And Saves response as "BoardObject"
     #Post Condition
```
Comment (Enrique-C) [inline trello/api/features/smoke/BoardSmoke.feature]: Delete this code line please

**Review by Enrique-C — APPROVED** [review]

**Review by Amet015 — APPROVED** [review]

**Pull request merged**